lib/toolbox_web/live/package_live.ex

```diff
@@ -62,13 +63,30 @@ defmodule ToolboxWeb.PackageLive do
       update_activity_if_outdated(package, github.sync_at)
     end
 
+    {related_packages, _} = Toolbox.Packages.list_packages_from_category(package.category, 5)
+
+    related_packages =
+      related_packages
+      |> Enum.reject(fn p -> p.name == name end)
```

Comment:
```suggestion
      |> Enum.reject(&(&1.name == name))
```
I think here we can align with how you were already matching by 1 attribute